airbyte-integrations/connectors/source-google-ads/source_google_ads/source.py

```diff
@@ -225,6 +225,8 @@ def check_connection(self, logger: logging.Logger, config: Mapping[str, Any]) ->
                         customer_id=customer.id,
                         login_customer_id=customer.login_customer_id,
                     )
+                except AirbyteTracedException as e:
+                    raise e
```

Comment:
I'm a little bit worried about losing the underlying context here, probably we should raise something like:

```python
raise e from e
```

But if it is OK, and we don't lose the context of the `AirbyteTracedException` raised prior, we should be fine with this.

Reply:
Thanks, updated